portfolio/src/main/webapp/index.html

```diff
@@ -8,11 +8,89 @@
   </head>
   <body>
     <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+      <div id="headline">
+        <div id="name">Hi there, I'm Shradha </div>
+        <span class="hand">👋</span> 
+      </div>
+      <br>
+      <img id="my-pic" src="/images/my_photo.jpeg" alt="My Picture."> 
+      <br><br>
+
+      <div id="about-me">
+        <p id="bio">
+          Born and raised in Haryana,
+          I am a final year Computer Engineering student from NSIT, Delhi.
+          My experience in tech includes working with Web Development, Linux OS and Machine Learning.
+          I also like working with neural networks and open source APIs.
+          In my college, I am a part of a technical society, taking classes for juniors to share my experience and knowledge of stuff like data structures, algorithms and web development.
+          Apart from that I have also been a member of the theatrical society to do stage plays.
+          For leisure, I enjoy watching a lot of TV series and movies.
+         <br>
+        </p>
+      </div>
+
+     <div id="projects">
+        <h1>Recent projects</h1>
+        <ul>
+            <li>
+              <b> Twitter Bot Detection </b> - Machine Learning(RVM)
+              <button onclick="addProjectDetail(this.id)" id="a">Details</button>
```

Comment:
Since you added a button here, should it be accessible as well? Set the `tabindex` property of the button tag. This will make it accessible by keyboard.